exonum-java-binding/CHANGELOG.md

```diff
@@ -21,8 +21,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 The main feature of the release is enhanced support of dynamic services and
 their lifetime. Service instances might be stopped and resumed now. While
-stopped, the service artifact can be upgraded, which allows services API and
-business logic updates.
+stopped, the service artifact can be upgraded, which allows services API
+updates.
```

Comment:
What is "services API updates"? We don't allow artifact upgrades at all. Replacing the artifact is not a supported operation (it is an undocumented hack).